seacrowd/sea_datasets/id_sentiment_analysis/id_sentiment_analysis.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = """\
+@misc{ridife2019idsa,
+  author = {Fe, Ridi},
+  title = {Indonesia Sentiment Analysis Dataset},
+  year = {2019},
+  publisher = {GitHub},
+  journal = {GitHub repository},
+  howpublished = {\url{https://github.com/ridife/dataset-idsa}}
```

Comment:
add a backlash as escape literal, otherwise runs into the following error 

```
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 189-190: truncated \uXXXX escape
```

In other words: `howpublished = {\\url{https://github.com/ridife/dataset-idsa}}`